src/settings.rs

```diff
@@ -17,6 +17,11 @@ static DEFAULT_PORT: u16 = 8000;
 // real `Settings`?
 //
 /// Configuration settings and options
+///
```

Comment:
I would like to include this change because [it is referenced](https://github.com/mozilla-services/contile/pull/278/files/0276a1b566be68fbbd3fe161e5c624d683d99a6a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R24) in the template info.

Reply:
That sounds reasonable! 👍🏻